setup.py

```diff
@@ -98,6 +84,7 @@
     platforms=['any'],
     packages=['cmd2'],
     keywords='command prompt console cmd',
+    python_requires='>=3.4',
```

Comment:
I figure we need some way of ensuring nobody tries to install the new version of ``cmd2`` on older un-supported versions of Python.  If we can make the assumption that people will have a pretty new version of setuptools >= 34.4 then this is a really easy way.  If we think it isn't wise to make that assumption, then we could remove this and do something more like:
```Python
import sys
if sys.version_info < (3,4):
    sys.exit('Sorry, Python < 3.4 is not supported by this version of cmd2.  Install cmd2 0.8.x instead')
```